### PR TITLE
Added plurals to the strings

### LINKED ISSUE #12451: May be plurals added to following strings
Running following lint give warning for the strings that should be changed to plurals.
```
lint --check PluralsCandidate ./res
```
http://tools.android.com/tips/lint-checks

Result
```
res\values\02-strings.xml:409: Warning: Formatting %d followed by words ("files"): This should probably be a plural rather than a string [PluralsCandidate]
    <string name="malicious_archive_exceeds_limit">Malicious archive. Size exceeds %1$s or contains more than %2$d files</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
res\values\03-dialogs.xml:91: Warning: Formatting %d followed by words ("existing"): This should probably be a plural rather than a string [PluralsCandidate]
    <string name="import_update_details">Updated %1$d of %2$d existing notes.</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
res\values\03-dialogs.xml:178: Warning: Formatting %d followed by words ("cards"): This should probably be a plural rather than a string [PluralsCandidate]
    <string name="integrity_check_fixed_no_home_deck">%d cards with incorrect home decks were recovered. Please see the manual for more information.</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
res\values\04-network.xml:56: Warning: Formatting %d followed by words ("seconds"): This should probably be a plural rather than a string [PluralsCandidate]
    <string name="sync_log_clocks_unsynchronized">Your clockÆs off by %1$d seconds%2$s. Make sure that the date, time, and timezone on your phone are set correctly, then sync again.</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
res\values\04-network.xml:83: Warning: Formatting %d followed by words ("media"): This should probably be a plural rather than a string [PluralsCandidate]
    <string name="sync_media_partial_updated">Media sync partially completed:\n%1$d media files downloaded.\n%2$d media files uploaded.</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
res\values\04-network.xml:84: Warning: Formatting %d followed by words ("media"): This should probably be a plural rather than a string [PluralsCandidate]
    <string name="sync_media_changes_count">%d media changes to upload</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
res\values\04-network.xml:85: Warning: Formatting %d followed by words ("media"): This should probably be a plural rather than a string [PluralsCandidate]
    <string name="sync_media_downloaded_count">%d media files downloaded</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
res\values\10-preferences.xml:126: Warning: Formatting %d followed by words ("cards"): This should probably be a plural rather than a string [PluralsCandidate]
    <string name="notification_minimum_cards_due">More than %d cards due</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
0 errors, 8 warnings
```

### PR DESCRIPTION
**Pull request by pratyaksh1610**
## Purpose / Description
Added plurals to the strings

## Fixes
Fixes #12451

### CODE REVIEW

**Comment by david-allison** [comment]
From CI:

```
PreferencesTest > fragmentsDoNotThrowOnCreation FAILED
    java.lang.NullPointerException
        at com.ichi2.anki.preferences.NotificationsSettingsFragment.updateNotificationPreference(NotificationsSettingsFragment.kt:72)
        at com.ichi2.anki.preferences.NotificationsSettingsFragment.initSubscreen(NotificationsSettingsFragment.kt:49)
        at com.ichi2.anki.preferences.SettingsFragment.onCreatePreferences(SettingsFragment.kt:56)
        at androidx.preference.PreferenceFragmentCompat.onCreate(PreferenceFragmentCompat.java:161)
        at androidx.fragment.app.Fragment.performCreate(Fragment.java:3090)
        at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:475)
        at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:257)
        at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1899)
        at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:1817)
        at androidx.fragment.app.FragmentManager.execSingleAction(FragmentManager.java:1729)
        at androidx.fragment.app.BackStackRecord.commitNow(BackStackRecord.java:317)
        at com.ichi2.anki.preferences.PreferencesTest.fragmentsDoNotThrowOnCreation$lambda$2(PreferencesTest.kt:103)
        at androidx.test.core.app.ActivityScenario.lambda$onActivity$2$ActivityScenario(ActivityScenario.java:660)
        at androidx.test.core.app.ActivityScenario$$Lambda$4.run(ActivityScenario.java:652)
        at androidx.test.core.app.ActivityScenario.onActivity(ActivityScenario.java:670)
        at com.ichi2.anki.preferences.PreferencesTest.fragmentsDoNotThrowOnCreation(PreferencesTest.kt:67)
```

File: AnkiDroid/src/main/res/values/02-strings.xml [hunk 1]
```diff
@@ -421,7 +421,10 @@
     <string name="not_valid_js_addon">%s is not a valid javascript addon package</string>
     <string name="could_not_create_dir">Could not create directory %s</string>
     <string name="malicious_archive_entry_outside">Malicious archive. Contains entry outside of the target dir: %s</string>
-    <string name="malicious_archive_exceeds_limit">Malicious archive. Size exceeds %1$s or contains more than %2$d files</string>
+    <plurals name="malicious_archive_exceeds_limit">
```
Comment (david-allison) [inline AnkiDroid/src/main/res/values/02-strings.xml:424]: I believe you want to change the key, so this updates the translation on CrowdIn

Comment (pratyaksh1610) [reply AnkiDroid/src/main/res/values/02-strings.xml:424]: How to achieve that ?

Comment (pratyaksh1610) [reply AnkiDroid/src/main/res/values/02-strings.xml:424]: @david-allison

Comment (mikehardy) [reply AnkiDroid/src/main/res/values/02-strings.xml:424]: You literally change the key name. That is: you alter the `name` attribute in the XML element to be something new. That way crowdin removes the previous translations (which were incorrect) and adds whole new ones (your new plurals)

**Comment by pratyaksh1610** [comment]
Please reopen this PR.

**Comment by mikehardy** [comment]
We can't, you have destroyed the commits on the branch then pushed 0 commits. There's no diff to base a PR on so github closes it. You'll have to resurrect it using old commits (like the reference left from your erroneous push) https://github.com/ankidroid/Anki-Android/commit/9c079fa4d0bad25b432df82b6afc7313d0c80d47

**Pull request closed without merging**